src/core/init/initialize_media_source.ts

```diff
@@ -392,30 +370,30 @@ export default function InitializeOnMediaSource(
         const reloadMediaSource$ = new Subject<{ currentTime : number;
                                                  isPaused : boolean; }>();
         const mediaSourceLoader$ = mediaSourceLoader(mediaSource, position, shouldPlay)
-          .pipe(tap(evt => {
+          .pipe(mergeMap((evt) => {
```

Comment:
What happened to the filterMap ?

Reply:
:D Nice!
It's exactly the right functional use case.
Will do

Reply:
Done

Reply:
Great